src/widgets/SelectStatus/SelectStatus.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC, memo } from 'react';
+import {
+    FormControl, InputLabel, MenuItem, Select, SelectChangeEvent,
+} from '@mui/material';
+import { StatusEnum } from '../../entities/Task';
+import { returnStatusColor } from '../../shared/lib/helpers';
+
+interface SelectStatusProps {
+    onChange?: (e: SelectChangeEvent) => void;
+    value: StatusEnum;
+    color?: 'success' | 'primary' | 'secondary'
+}
+
+export const SelectStatus: FC<SelectStatusProps> = memo(({ onChange, value, color = 'primary' }) => (
```

Comment:
`color` is not used and seems can be removed from props

Reply:
Fixed